src/App.vue

```diff
@@ -18,11 +20,13 @@ export default {
   name: "MainComponents",
   data() {
     return {
-      username: localStorage.getItem("token"),
+      username: localStorage.getItem("token").toUpperCase(),
     };
   },
   created() {
     this.$store.dispatch("loadToken");
+    this.$store.commit("setCreateMode", false);
+    this.$store.commit("setLocations", []);
   },
```

Comment:
@JatinThakur14 Is this needed? I think locations and create mode will have default values when the store is initialized, in their definitions itself